README.md

```diff
@@ -125,22 +125,22 @@ SQLx is compatible with the [`async-std`], [`tokio`], and [`actix`] runtimes; an
 # PICK ONE OF THE FOLLOWING:
 
 # tokio (no TLS)
-sqlx = { version = "0.7", features = [ "runtime-tokio" ] }
+sqlx = { version = "8", features = [ "runtime-tokio" ] }
```

Comment:
```suggestion
sqlx = { version = "0.8", features = [ "runtime-tokio" ] }
```

Reply:
Thanks for your observation, I made the correction.